<?xml version="1.0"?>
<launch>
  <arg name="display" default="true"/>

  <arg name="sensor"        default="camera"/>
  <arg name="tf_prefix"     default="$(arg sensor)"/>

  <arg name="service"       default="false"/>
  <arg name="markers"       default="markers"/>

  <arg name="image"         default="color/image_raw"/>
  <arg name="depth_image"   default="aligned_depth_to_color/image_raw"/>
  <arg name="camera_info"   default="aligned_depth_to_color/camera_info"/>

  <arg name="dictionary" default="3"/>
  <arg name="centre"     default="100"/>
  <arg name="top_left"   default="91"/>
  <arg name="top_right"  default="92"/>
  <arg name="bot_right"  default="108"/>
  <arg name="bot_left"   default="107"/>

  <arg name="is_depth_in_meters" default="false"/>

  <include file="$(find aruco_detector)/launch/base_marker.launch" pass_all_args="true">
    <arg name="marker_type"   value="diamond"/>
    <arg name="sensor_type"   value="depth"/>
  </include>
</launch>

